Model/Event/Event.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Model.DTO;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Model
+{
+    public class Event : DomainObject
+    {
+        #region props
+
+        public string Title { get; set; }
+
+        public string Description { get; set; }
+
+        public string Address { get; set; }
+
+        public DateTime EventDate { get; set; }
+
+        public TimeSpan EventDuration { get; set; } 
+
+        public Image Image { get; set; }
+
+        public long Image_Id { get; set; }
```

Comment:
```suggestion
        public long Image_Id { get => Image.Id ; set => Image.Id = value; }
```
Можем так сделать? Если да, то поправь остальные связанные места в классе